src/inmanta/agent/handler.py

```diff
@@ -205,8 +205,48 @@ def _log_msg(
         raise NotImplementedError
 
 
+# TODO: extract logging behavior from HandlerContext
+class HandlerLogger(LoggerABC):
```

Comment:
I vaguely recall that we didn't need this anymore? i.e. I think all changes in this file can be reverted, unless you see something we want to keep?